### PR TITLE
Net 590

### PR DESCRIPTION
**Pull request by mattkasun**
add a brief guide to using netmaker from the command line (without the UI)

### CODE REVIEW

File: nmclt-standalone.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        
+            export NN_DOMAIN=example.com
+
+******************
+Set Context
+******************
+.. code-block::
+            
+            nmctl context set commandline --endpoint https://api.$NM_DOMAIN --master_key $MASTERKEY
+            nmctl context use commandline
```
Comment (abhishek9686) [inline nmclt-standalone.rst:45]: why use master_key?
docs should suggest user to use their username and password, rather than master key

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: nmclt-standalone.rst (hunk 1 above)
Comment (mattkasun) [reply nmclt-standalone.rst:45]: because user hasn't been created yet so have to use masterkey

Comment (abhishek9686) [reply nmclt-standalone.rst:45]: first user (super admin) can be created without any auth

Comment (abhishek9686) [reply nmclt-standalone.rst:44]: if all users are expected to use master key what's point of RBAC on server then ?
master key's main intention  is only to recover the server, if when say super admin forgets his password for example

Comment (mattkasun) [reply nmclt-standalone.rst:44]: cannot set context without either masterkey or username/password .... if set to username/password user create will fail as user does not exist.

added a step to reset the context

Comment (abhishek9686) [reply nmclt-standalone.rst:44]: yes, the idea is superadmin is the one who will create other users, superadmin is the first user who will be created on the server, so don't have to use masterkey

Comment (mattkasun) [reply nmclt-standalone.rst:44]: either  masterkey or username/password is required to set context

Comment (abhishek9686) [reply nmclt-standalone.rst:44]: users will have their usernames and password to set context, all the users are not expected to have master key with them

Comment (mattkasun) [reply nmclt-standalone.rst]: added clarification that initial admin must be created by superAdmin

**Review by Aceix — APPROVED** [review]

**Pull request merged**